Bottom: 811e03ecb2557253e4d730b9dcd0428ce62ae4d7
Top:    7142312d69a8a3a5fdaedfbf53dba3da85536cfc
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-20 09:59:30 +1300

Refresh of stream-to-dict

---

diff --git a/.travis.yml b/.travis.yml
index 1bcce89..2a39c55 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -8,7 +8,7 @@ python:
   - "pypy"
 
 install:
-  - pip install -q --use-mirrors fixtures extras
+  - pip install -q --use-mirrors fixtures extras python-mimeparse
   - python setup.py -q install
 
 script:
